Mobile/starter_project_matador/lib/features/example/presentation/screen/splash_screen.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:matador/features/example/presentation/widgets/background_container.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  static const Color backgroundColor = Color.fromARGB(125, 244, 247, 255);
```

Comment:
avoid defining colors here. use something like this https://flutter-starter.geekyants.com/docs/add-color-constants